### PR TITLE
Fix notes and warnings

### PR DESCRIPTION
**Pull request by cicdguy**
- Import and export proper functions
- Title should be in `title` case

### CODE REVIEW

File: R/scale_nest_discrete.R [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+#'
+#' @description `r lifecycle::badge("stable")`
+#'
+#' Adds scaling manually or some such thing. IDK
+#' what this does TBH, I'm just trying to
+#' appease R CMD check.
+#'
+#' @param palette (``)
+#' @param palette (``) 
+#' @importFrom ggplot2 waiver discrete_scale ScaleDiscrete
```
Comment (cicdguy) [inline R/scale_nest_discrete.R:11]: Yo @edelarua - would you be able to add docs to these functions? You can use this branch to update.

Comment (cicdguy) [reply R/scale_nest_discrete.R:11]: And unit tests for this. Otherwise these need to be removed as YAGNI

**Comment by edelarua** [comment]
@gogonzo I noticed that these two functions `scale_nest_manual` and `scale_nest_discrete` were added by you to `nestcolor` several weeks ago but are not currently used anywhere. Dinakar would like to remove/internalize unused functions unless we plan to implement them soon. Are there any plans to use these two functions in the future? If so we can add in the appropriate documentation and unit tests.

**Comment by cicdguy** [comment]
> @gogonzo I noticed that these two functions `scale_nest_manual` and `scale_nest_discrete` were added by you to `nestcolor` several weeks ago but are not currently used anywhere. Dinakar would like to remove/internalize unused functions unless we plan to implement them soon. Are there any plans to use these two functions in the future? If so we can add in the appropriate documentation and unit tests.

FYI @gogonzo we are trying to improve code coverage as we will be submitting these packages for validation soon, hence the ask for either adding docs + unit tests or completely removing them if they aren't needed.

**Comment by gogonzo** [comment]
Hi @edelarua @cicdguy  I'm happy to remove these functions if they are not used. I don't think we will decide to implement them ever as nestcolors can't be in dependencies of our open source packages. nestcolor can be released internally but will never be accepted on CRAN.

**Comment by edelarua** [comment]
> Hi @edelarua @cicdguy I'm happy to remove these functions if they are not used. I don't think we will decide to implement them ever as nestcolors can't be in dependencies of our open source packages. nestcolor can be released internally but will never be accepted on CRAN.

Thanks @gogonzo! I will remove the functions for now.

**Pull request merged**